### PR TITLE
TableStoreWriteLock rename exception handling method

### PR DESCRIPTION
**Pull request by tgd**
Resolve spurious test failure - see #1431 for more details.

### CODE REVIEW

File: src/main/java/net/openhft/chronicle/queue/impl/single/TableStoreWriteLock.java [hunk 1]
```diff
@@ -96,7 +96,7 @@ private void lockAssertPostConditions() {
                 ((lockedByThread = Thread.currentThread()) != null && (lockedHere = new StackTrace()) != null);
     }
 
-    private void lockHandleTimeoutException(long currentLockValue) {
+    private void lockHandleTimeout(long currentLockValue) {
```
Comment (alamar) [inline src/main/java/net/openhft/chronicle/queue/impl/single/TableStoreWriteLock.java]: Perhaps add some verb to method name?

**Review by alamar — APPROVED** [review]

File: src/main/java/net/openhft/chronicle/queue/impl/single/TableStoreWriteLock.java (hunk 1 above)
Comment (tgd) [reply src/main/java/net/openhft/chronicle/queue/impl/single/TableStoreWriteLock.java]: How about `handleLockTimeout`?

Comment (alamar) [reply src/main/java/net/openhft/chronicle/queue/impl/single/TableStoreWriteLock.java]: Looks OK if it actually does that!

Comment (tgd) [reply src/main/java/net/openhft/chronicle/queue/impl/single/TableStoreWriteLock.java]: It does. I tweaked the naming a little bit more - I also removed the `Exception` substring from another method!

**Review by alamar — APPROVED** [review]

**Pull request merged**